index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Mono:ital,wght@1,700&display=swap" rel="stylesheet">
+</head>
+<body>
+      <nav>
+          <p class="logo">My Logo</p>
+          <ul>
+            <li><a href="#">portfolio</a></li>
+            <li><a href="#">about</a></li>
+            <li><a href="#">contact</a></li>
+          </ul>
+          <div class="menu">
+            <img src="./images/Icon - Menu.png" alt="">
+            <a href="toogle-nav">Menu</a>
```

Comment:
- [ ] Well done  👏😎 the menu icon is displayed now, However, the design is still inconsistent with the project requirement, the hamburger menu icon should be a clickable element and to do so Please follow the example below:

```html
<div class="menu_wrapper_class">
    <a href="menu_link_class">
         <img src="hamburger_icon_src" alt="menu">
     </a>
</div>
```

### Your Design
![image](https://user-images.githubusercontent.com/68030297/219718626-3d8f2ca7-84cd-4244-b091-b79abb9f2bf5.png)

### Figma Design
![image](https://user-images.githubusercontent.com/68030297/219719387-45070100-836d-4a61-b24c-8846f8c10d3b.png)